emily/handler/src/database/entries/deposit.rs

```diff
@@ -694,6 +694,8 @@ impl ValidatedUpdateDepositsRequest {
             .map(|(_, update)| Chainstate {
                 stacks_block_hash: update.event.stacks_block_hash,
                 stacks_block_height: update.event.stacks_block_height,
+                bitcoin_block_hash: Default::default(),
```

Comment:
qq: what implications does setting this to `default()` have? i would have expected this to be the bitcoin chain tip at the time of event creation, but i'm less familiar with the exact constraints of emily's chainstate management so i'm not really sure here.

Same Q for withdrawals.